src/js/boot.js.tpl

```diff
@@ -34,7 +34,9 @@ define([], function() {
                   ["world/ng-interactive/2017/jun/16/chalk-girl-hong-kong-democracy-umbrella-movement-protester-china", "the-infamous-chalk-girl"],
                   ["film/ng-interactive/2017/jun/30/fish-story-anglesey-video-fish-surnames", "fish-story"],
                   ["global/ng-interactive/2017/jun/28/pitching-up-documentary", "pitching-up"],
-                  ["us-news/ng-interactive/2017/jul/17/dearborn-michigan", "dearborn-michigan"]
+                  ["us-news/ng-interactive/2017/jul/17/dearborn-michigan", "dearborn-michigan"],
```

Comment:
do we want to update the dearborn path to the live path`us-news/ng-interactive/2017/aug/04/dearborn-michigan-divided-muslim-american-donald-trump-documentary` too?